### DISCUSSION TITLE
Error 1067 while trying to work with Certificate on windows

### DISCUSSION THREAD

**Discussion by mrwegle** [drakkan/sftpgo#1300] [General]
Hi I'm trying to work with my certificate on Windows.

I created httpd.env with those values:

SFTPGO_HTTPD__BINDINGS__0__PORT=443
SFTPGO_HTTPD__BINDINGS__0__ENABLE_HTTPS=1
SFTPGO_HTTPD__BINDINGS__0__CERTIFICATE_FILE="mypath/mycert.crt"
SFTPGO_HTTPD__BINDINGS__0__CERTIFICATE_KEY_FILE="my_private_key.key"

When I add last line and try to restart service it doesn't work and show error 1067.

I used your guide for Linux, so I assumed syntax in httpd.env will be same and file extensions need to be same. Am I wrong?

**Comment by drakkan** [comment]
Hello, the procedure  is the same, the certificates must be in pem format. Try to better understand the issue by reading the logs. I'm sorry but I no longer provide free support.

**Reply by wazze** [reply]
Hi,
Ensure the path of the certificates is using double backslash "\\\\" in the json config file....this solved the very same issue when I encountered it.
sftpgo error log " Windows service failed to start, error: While parsing config: invalid character 'P' in string escape code" / win event viewer  error 1067
